blobfuse/blobfuse.cpp

```diff
@@ -248,6 +255,18 @@ int read_config(const std::string configFile)
             std::string resourceIdStr(value);
             config_options.resourceId = resourceIdStr;
         }
+        else if(line.find("caCertFile") != std::string::npos)
+        {
+            syslog(LOG_DEBUG, "caCertFile has a non null value");
+            std::string caCertFileStr(value);
+            config_options.caCertFile = caCertFileStr;
```

Comment:
We shall have a debug log here as well, stating we have hit this condition and using a particular file for cert